docs/integrating-scrapy.md

```diff
@@ -140,6 +140,58 @@ The [scheduler](https://docs.scrapy.org/en/latest/topics/scheduler.html) is a co
 
 Another default Scrapy [downloader middleware](https://docs.scrapy.org/en/latest/topics/downloader-middleware.html) that requires replacement is [HttpProxyMiddleware](https://docs.scrapy.org/en/latest/topics/downloader-middleware.html#module-scrapy.downloadermiddlewares.httpproxy). To utilize the use of proxies managed through the Apify [ProxyConfiguration](https://github.com/apify/apify-sdk-python/blob/master/src/apify/proxy_configuration.py), we provide [ApifyHttpProxyMiddleware](https://github.com/apify/apify-sdk-python/blob/master/src/apify/scrapy/middlewares/apify_proxy.py). When using the Apify CLI wrapping tool, the default `HttpProxyMiddleware` is disabled, and [ApifyHttpProxyMiddleware](https://github.com/apify/apify-sdk-python/blob/master/src/apify/scrapy/middlewares/apify_proxy.py) takes its place. Additionally, inspect the [.actor/input_schema.json](https://github.com/apify/actor-templates/blob/master/templates/python-scrapy/.actor/input_schema.json) file, where proxy configuration is specified as an input property for your Actor. The processing of this input is carried out together with the middleware configuration in [src/main.py](https://github.com/apify/actor-templates/blob/master/templates/python-scrapy/src/main.py).
 
+## Known limitations
+
+There are some known limitations of running the Scrapy projects on Apify platform we are aware of.
+
+### Asynchronous code in spiders and other components
+
+Scrapy asynchronous execution is based on the [Twisted](https://twisted.org/) library, not the
+[AsyncIO](https://docs.python.org/3/library/asyncio.html), which brings some complications on the table.
+
+Due to the asynchronous nature of the Actors, all of their code is executed as a coroutine inside the `asyncio.run`.
+In order to execute Scrapy code inside an Actor, following the section
+[Run Scrapy from a script](https://docs.scrapy.org/en/latest/topics/practices.html?highlight=CrawlerProcess#run-scrapy-from-a-script)
+from the official Scrapy documentation, we need to invoke a
+[CrawlProcess.start](https://github.com/scrapy/scrapy/blob/2.11.0/scrapy/crawler.py#L393:L427)
+method. This method triggers Twisted's event loop, also known as a reactor.
+Consequently, Twisted's event loop is executed within AsyncIO's event loop.
+On top of that, when employing AsyncIO code in Spiders or other components, it necessitates the creation of a new
```

Comment:
Is there any particular reason why spiders is capitalized here?

Reply:
it's a mistake, thanks